forbiditerative/plan.py

```diff
@@ -281,7 +277,14 @@ def validate_input(args):
         title="time and memory limits")
     lim.add_argument("--overall-time-limit")
     lim.add_argument("--overall-memory-limit")
- 
+
```

Comment:
I wasn't sufficiently clear in my previous comments, I apologize. 
I wanted to ask whether your meaning with this argument is the same as what I mentioned. If yes, then you have implemented a mechanism that makes the build_args() in planner_call.py redundant and we could remove those.

Reply:
Or I can do that after we merge this PR.

Reply:
I actually have no idea if it makes `build_args()` redundant or not, because that function is never overridden in any of the `PlannerCall` implementations.

My assumption is that `build_args()` is for when you want a specific implementation of `ReformulationPlannerCall` or `PlansToJsonPlannerCall` to append to the callstring for whatever reason.

You could remove the dead code, or you could assume someone had something in mind.

Reply:
That someone was me and the build_args() is there from the times when different planners were located in their code bases, and required different builds. Since then, I have integrated all relevant planners into this codebase and they all use the same build.